wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileActionsFactory.swift

```diff
@@ -200,7 +200,8 @@ final class ProfileActionsFactory {
             }
 
             // If the user is not from the same team as the other user, allow blocking
-            if user.isConnected && !isOnSameTeam && !user.isWirelessUser && !user.hasIncompleteMetadata {
+
+            if user.isConnected && !isOnSameTeam && !user.isWirelessUser && !user.hasEmptyName {
```

Comment:
is hasEmptyName a stored property or a computed one?

Reply:
`hasEmptyName` is computed property. It's a helper to reduce repeated code
```
 public var hasEmptyName: Bool {
        guard let name = name else {
            return true
        }
        return name.isEmpty
    }
```